demo/index.html

```diff
@@ -113,6 +119,7 @@
       window.setTimeout(function() {
         var t = document.querySelector('#delayedLoader');
         t.libraryUrl = 'https://apis.google.com/js/drive-realtime.js?onload=%%callback%%';
+        t.parentNode.libraryUrl = t.libraryUrl;
```

Comment:
I should put a comment. This is necessary for the demo to work in 2.x since in 1.x you set values on the `template[is="dom-bind"]` as opposed to 2.x where you set the values on `dom-bind`